Dockerfile

```diff
@@ -0,0 +1,23 @@
+# ---------------------- Builder image ----------------------
+
+FROM node:12-slim AS builder
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm i
+
+RUN npm run build
+
+RUN rm -rf src/
+
+# ---------------------- Runtime image ----------------------
+
+FROM node:12-slim
```

Comment:
The original runtime image is 
node:10.20.1-alpine3.11

Reply:
Have updated runtime image to "node:10.20.1-alpine3.11"